x-pack/plugins/infra/public/pages/metrics/hosts/components/controls_content.tsx

```diff
@@ -59,7 +59,7 @@ export const ControlsContent: React.FC<Props> = ({
 
   return (
     <LazyControlsRenderer
-      getCreationOptions={async ({ addDataControlFromField }) => ({
+      getInitialInput={async () => ({
```

Comment:
I like that I don't need `addDataControlFromField`now.
NIT - just a quick thought about the return value - maybe it would be nice also to expect the non-promise version of the input, but it's not something blocking.